modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -352,6 +358,117 @@ public ObjectEntry postScopeScopeKey(
 			scopeKey);
 	}
 
+	@Override
+	public ValidateResult postScopeScopeKeyValidate(
+			String scopeKey, ValidateRequest validateRequest)
+		throws Exception {
+
+		if (!FeatureFlagManagerUtil.isEnabled(
+				contextCompany.getCompanyId(), "LPD-37056")) {
+
+			throw new UnsupportedOperationException();
+		}
+
+		ObjectEntryManager objectEntryManager =
+			_objectEntryManagerRegistry.getObjectEntryManager(
+				_objectDefinition.getStorageType());
+
+		try {
+			objectEntryManager.validateObjectEntry(
+				_getDTOConverterContext(null), _objectDefinition,
+				validateRequest.getValues(),
+				Arrays.asList(validateRequest.getValidationKeys()), scopeKey);
+		}
+		catch (ObjectValidationRuleEngineException
+					objectValidationRuleEngineException) {
+
+			return new ValidateResult() {
+				{
+					setErrors(
+						() -> transformToArray(
+							objectValidationRuleEngineException.
+								getObjectValidationRuleResults(),
+							objectValidationRuleResult -> new Error() {
+								{
+									setErrorMessage(
+										objectValidationRuleResult::
+											getErrorMessage);
+									setObjectFieldName(
+										objectValidationRuleResult::
+											getObjectFieldName);
+									setValidationKey(
+										objectValidationRuleResult::
+											getValidationKey);
+								}
+							},
+							Error.class));
+					setSuccess(() -> false);
+				}
+			};
+		}
+
+		return new ValidateResult() {
+			{
+				setSuccess(() -> true);
+			}
+		};
+	}
+
+	@Override
+	public ValidateResult postValidate(ValidateRequest validateRequest)
```

Comment:
If I understand correctly, the code for both methods (`postScopeScopeKeyValidate` and `postValidate`) are very similar, the only difference is that one passes the `scopeKey` value to the `objectEntryManager.validateObjectEntry` method and the other passes the null value. 
Can we avoid this code duplication by creating a private method to reuse all this logic?